bin/cluster/management_update

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+Script to call when there is a change in the management state
+
+=head1 SYNOPSIS
+
+bin/cluster/management_update <change_type> <vrrp_ip> <state> <priority>
+
+=head1 DESCRIPTION
+
+Made for keepalived so the pf services can be adjusted depending of the running node
+
+=cut
+
+use constant INSTALL_DIR => '/usr/local/pf';
+use lib INSTALL_DIR . "/lib";
+
+use pf::log;
+
+my $logger = get_logger;
+
+$logger->info("Refreshing services for management mode");
+
+my $vrrp_ip = $ARGV[1];
+my $state = $ARGV[2];
+my $priority = $ARGV[3];
+
+if($state eq "MASTER"){
+    $logger->info("Transition to master state, starting management services.");
+    `/usr/local/pf/bin/pfcmd service pfdhcplistener start`;
```

Comment:
Just a question, before we sent a kill -HUP to pfdhcplistener to reload his config, the goal was to keep pfdhcplistener running on all the member and just change on the fly if the pfdhcplistener must capture traffic or no.
I just prefer to have all the services running on all the members instead of trying to start/stop each service.
The other question, is what happen if i stop the reg interface on the master server, will it stop the pfdhcplistener for all the interfaces ?